OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectWorld.cs

```diff
@@ -48,6 +49,9 @@ class DreamMetaObjectWorld : DreamMetaObjectRoot {
                     _cfg.SetCVar(CVars.NetTickrate, variableValue.GetValueAsInteger()); break;
                 case "maxz":
                     _dreamMapManager.SetZLevels(variableValue.GetValueAsInteger()); break;
+                case "log":
+                    _dreamManager.WorldLog = new LogOutputResource(GetLogPath(variableValue));
+                    break;
```

Comment:
Running this code in byond shows that setting `world.log` to a string causes it to actually be set to a resource
```
/mob/verb/test()
    world.log = "output.txt"
    usr << istext(world.log)
    usr << isfile(world.log)
```
Output:
```
0
1
```
Instead of having `_dreamManager.WorldLog` always be a special `LogOutputResource` it'd likely be better as a more generic `DreamResource` (keeping something akin to the `ConsoleOutputResource` that currently exists). You could return this in the getter for `world.log` which would remove the need for the special handling in the `BitShiftLeft` opcode.

Doing `world.log = null` also appears to set it back to the console.